src/wrap-store/wrapStore.js

```diff
@@ -19,8 +19,9 @@ const promiseResponder = (dispatchResult, send) => {
       });
     })
     .catch((err) => {
+      console.error(err);
       send({
-        error: err,
+        error: err.message,
```

Comment:
This was my oversight in the beginning. Because this is sending the payload to the "frontend", it doesn't serialize a complex object. So an error object would just show up as `[object object]` when inspect the popup/content page.